demo/demo_2D_shepp_logan.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+import svmbir
+
+num_rows = 256
+num_cols = num_rows
+num_views = 144
+threshold = 0.05
+
```

Comment:
I would include an overview of this file.  Something like
"""
This file demonstrates the generation of a Shepp-Logan phantom followed by sinogram projection and reconstruction using MBIR. 
"""